localstack/services/awslambda/provider.py

```diff
@@ -220,15 +221,15 @@ def _get_function_version(
         # TODO what if version is missing?
         return version
 
-    def _publish_version(
+    def _create_version_model(
         self,
         function_name: str,
         region: str,
         account_id: str,
         description: str | None = None,
         revision_id: str | None = None,
         code_sha256: str | None = None,
-    ):
+    ) -> tuple[FunctionVersion, bool]:
```

Comment:
would be great if we could add a docstring here documenting especially what the returned bool is used for